pages/common.py

```diff
@@ -130,7 +130,27 @@ def body(content):
 <script src="assets/js/custom.js"></script>
 <!-- SLICK CAROUSEL -->
 <script type="text/javascript" src="assets/slick/slick.min.js"></script>
-
+<!-- BOOTBOX -->
+<script type="text/javascript" src="assets/js/bootbox.min.js"></script>
+
+<!-- Chat link modal alert handler, this is the only place it can go at the moment  -->
+<script type="text/javascript">
+jQuery(document).on("click", '[title="chatlink"]', function(e) {
+    bootbox.dialog({
+        message: "This chat is staffed by volunteers, and it may be that no one is around right now. Please ask your question, and someone will answer within several hours.",
```

Comment:
These user facing strings should be translatable substitution tokens and be included in the substitute call [like these](https://github.com/freenet/website/blob/335a686e7c242ad1a5d6203cb03bcae0f9cc7b9c/pages/index.py#L219).